airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -408,6 +412,15 @@ private Job getJob(DSLContext ctx, long jobId) {
         .flatMap(r -> getJobOptional(ctx, r.get("job_id", Long.class))));
   }
 
+  @Override
+  public List<Job> listJobs(ConfigType configType, Instant attemptEndedAtTimestamp) throws IOException {
+    final LocalDateTime timeConvertedIntoLocalDateTime = LocalDateTime.ofInstant(attemptEndedAtTimestamp, ZoneOffset.UTC);
+    return database.query(ctx -> getJobsFromResult(ctx
+        .fetch(BASE_JOB_SELECT_AND_JOIN + "WHERE " +
+            "CAST(config_type AS VARCHAR) IN " + Sqls.toSqlInFragment(Sets.newHashSet(configType)) + " AND " +
+            " attempts.ended_at > ? ORDER BY jobs.created_at ASC, attempts.created_at ASC", timeConvertedIntoLocalDateTime)));
```

Comment:
I think you can make your life a little easier here. In the other `listJobs` method in this class the order by query is already there as a constant. See:

https://github.com/airbytehq/airbyte/blob/master/airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java#L332-L340